glTF-Toolkit/src/GLTFLODUtils.cpp

```diff
@@ -16,6 +16,8 @@
 #include <algorithm>
 #include <iostream>
 #include <set>
+#include <codecvt>

```

Comment:
\<codecvt\> is deprecated: www.open-std.org/jtc1/sc22/wg21/docs/papers/2017/p0618r0.html

Reply:
In general I've been using the naïve conversion from wstring to string elsewhere. I don't have a great solution for this - it's unfortunate that the GLTF SDK works with std::string everywhere, while files can have non-ansi chars.
If this conversion is the best, we should put it in a helper function and use it everywhere (probably with a unit test to ensure it does what we want)

Reply:
Given that the GLTF spec states that "JSON must use UTF-8 encoding without BOM." my patch assumes that the std::string entries are UTF-8 encoded. I think that using codecvt way is better than the naïve approach, and agree that unit-testing this would be good.

Not sure about the timeframe for getting a c++ standard replacement for codecvt, another option would be to include a third-party lib e.g libiconv.